rust-toolchain

```diff
@@ -1 +1 @@
-stable
+nightly
```

Comment:
What do we need from `nightly`?

Reply:
That's for cargo-fuzz:

> Note: libFuzzer needs LLVM sanitizer support, so this only works on x86-64 Linux, x86-64 macOS and Apple-Silicon (aarch64) macOS for now. This also needs a nightly compiler since it uses some unstable command-line flags. You'll also need a C++ compiler with C++11 support.